course05/homework01/freekpaans/string.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <string>
+#include <ostream>
+#include <istream>
+
+class String
+{
+public:
+	String();
+	String(std::string);
+
+	bool Empty() const;
+	std::size_t Size() const;
+
+	bool operator==(const String) const; // These two copy the comparand, but we didn't cover references yet, right :)
+	bool operator!=(const String) const;
+
+	String Substring(int start) const;
+	String Substring(int start, int length) const;
+
+	const char* begin() const
+	{
+		return mChars;
+	}
+
+	const char* end() const
+	{
+		return mChars+mSize;
+	}
+
+	const static int MaxChars = 256;
+
+private:
+	String(const char* b, const char* e);
+	size_t mSize{};
```

Comment:
It's correct, but in general I would advice to write `= 0;` - I know that you are trying to use the uniform initialization which is definitely a great practice, but some more inexperienced developers might find this confusing / not clear.